src/cli/mod.rs

```diff
@@ -160,6 +192,36 @@ impl CkbArgs {
     pub fn client(&self) -> CkbRpcClient {
         CkbRpcClient::new(self.ckb_endpoint.as_str())
     }
+
+    pub fn fee_rate(&self) -> Result<u64> {
+        let value = if let Some(fixed) = self.fee_rate.value() {
+            log::info!("CKB fee rate: {} (fixed)", FeeRate(fixed));
+            fixed
+        } else {
+            let dynamic = self.client().dynamic_fee_rate()?;
+            log::info!("CKB fee rate: {} (dynamic)", FeeRate(dynamic));
+            if dynamic > 1_0000_0000 {
```

Comment:
In the beginning of 2024, there was an instance where the fee rate exceeded 1_0000_0000.
According to my recollection, the fee rate reached as high as 20_0000_0000+.

Reply:
![image](https://github.com/ckb-cell/ckb-bitcoin-spv-service/assets/1297478/31d202fa-4a20-4824-a190-64afc22e51e4)